packages/react/src/components/container/Container.tsx

```diff
@@ -6,11 +6,11 @@ import '../../styles/base.css';
 import styles from './Container.module.scss';
 import classNames from '../../utils/classNames';
 
-export type ContainerProps = React.PropsWithChildren<React.HTMLProps<HTMLDivElement>>;
+export type ContainerProps = React.PropsWithChildren<React.HTMLProps<HTMLDivElement>> & { alignWithHeader?: boolean };
```

Comment:
Maybe we should add JSDoc here about the prop? Something like "Align element's content with HDS Header's content." ?

Reply:
Yes, good point. Added!